### PR TITLE
Redis multi put

### PR DESCRIPTION
**Pull request by moritastic**

### CODE REVIEW

File: backends/aerospike.go [hunk 1]
```diff
@@ -50,22 +50,29 @@ func (a *Aerospike) Get(ctx context.Context, key string) (string, error) {
...
+			ttlSeconds = a.cfg.DefaultTTL
+		}
+		bins := as.BinMap{
+			binValue: value,
+		}
+		err = a.client.Put(&as.WritePolicy{
+			Expiration: uint32(ttlSeconds),
+		}, asKey, bins)
+		if err != nil {
+			return err
```
Comment (egreaves) [inline backends/aerospike.go:73]: Pardon my ignorance, but I noticed that the only return statement in the for loop is for the error case. What happens if there's an error before the last iteration of the for loop? Does execution continue for the other put requests?

File: backends/aerospike.go [hunk 2]
```diff
@@ -50,22 +50,29 @@ func (a *Aerospike) Get(ctx context.Context, key string) (string, error) {
...
-	}
-	bins := as.BinMap{
-		binValue: value,
-	}
-	err = a.client.Put(&as.WritePolicy{
-		Expiration: uint32(ttlSeconds),
-	}, asKey, bins)
-	if err != nil {
-		return err
+func (a *Aerospike) MultiPut(ctx context.Context, payloads []Payload) error {
```
Comment (egreaves) [inline backends/aerospike.go:53]: Would it be beneficial to maintain existing behavior by keeping `Put` around, passing in an array with a single payload to `MultiPut`?

File: endpoints/put.go [hunk 3]
```diff
@@ -56,6 +56,7 @@ func NewPutHandler(backend backends.Backend, maxNumValues int, allowKeys bool) f
 		resps := putResponsePool.Get().(PutResponse)
 		resps.Responses = make([]PutResponseObject, len(put.Puts))
 		defer putResponsePool.Put(resps)
+		payloads := []backends.Payload{}
```
Comment (egreaves) [inline endpoints/put.go:59]: This answers my question above.

File: backends/aerospike.go (hunk 1 above)
Comment (moritastic) [reply backends/aerospike.go:73]: It errors out if a put fails and does not continue processing the other requests. This is the same as the previous behavior.

**Comment by moritastic** [comment]
Going to do a PR into a "versioned" branch

**Pull request closed without merging**